libsolidity/ast/ASTJsonConverter.h

```diff
@@ -161,6 +168,7 @@ class ASTJsonConverter: public ASTConstVisitor
 		m_jsonNodePtrs.pop();
 	}
 
+	bool inEvent = false;
```

Comment:
Usually, members should have a `m_` prefix, not sure why `processed` does not have one.